ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -141,207 +141,203 @@ def check_if_proxies_work(logger):
         proxies_tested = 0
         # try put/get/verify on all available proxies on the local node
         for service in ServiceHelper.get_services():
-            if service.storagerouter_guid == AlbaHealthCheck.MACHINE_DETAILS.guid and service.type.name == ServiceType.SERVICE_TYPES.ALBA_PROXY:
```

Comment:
General remarks:
1) You committed the file with CRLF io LF
2) You have unused imports